crates/cli/src/cli/mod.rs

```diff
@@ -221,10 +221,9 @@ pub struct StartSimnet {
     /// Start surfpool with some CI adequate settings  (eg. surfpool start --ci)
     #[clap(long = "ci", action=ArgAction::SetTrue, default_value = "false")]
     pub ci: bool,
-    /// Apply suggested defaults for runbook generation and execution.
-    /// This includes executing any deployment runbooks, and generating in-memory deployment runbooks if none exist. (eg. surfpool start --autopilot)
-    #[clap(long = "autopilot", action=ArgAction::SetTrue, default_value = "false")]
-    pub autopilot: bool,
+    /// Apply suggested defaults for runbook generation and execution when running as part of an anchor test suite (eg. surfpool start --anchor-compatibility)
+    #[clap(long = "anchor-compatibility", action=ArgAction::SetTrue, default_value = "false")]
```

Comment:
`anchor-compatibility` or `anchor-legacy` ?
I know it's weird, but ideally we only enable this mode for anchor `0.x`